### PR TITLE
Week2 Homework 

### PR DESCRIPTION
**Pull request by gulyener**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -30,17 +36,91 @@
...
+    addRow(tbody, 'Updated', new Date(repo.updated_at).toLocaleString('en-GB'));
   }
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
-  window.onload = () => main(HYF_REPOS_URL);
+  function renderContributions(repo, ul) {
+    fetchJSON(repo.contributors_url)
+      .then(contributors => {
+        ul.innerHTML = '';
+        createAndAppend('h3', ul, { text: `Contributors:` });
```
Comment (remarcmij) [inline homework/index.js]: I think I already gave the feedback in week 1 that it is invalid to make an `h3` a child of an `li`.

File: homework/index.js [hunk 2]
```diff
@@ -30,17 +36,91 @@
...
+
+    fetchJSON(url)
+      .then(repositories => {
+        createOptionElements(repositories, select);
+
+        function setDefault() {
+          renderRepos(repositories[0], tableContainer);
+          renderContributions(repositories[0], ul);
+        }
+        setDefault();
```
Comment (remarcmij) [inline homework/index.js]: See my comments on this useless function in the week 1 feedback.

File: homework/index.js [hunk 3]
```diff
@@ -1,19 +1,25 @@
+/* eslint-disable prefer-destructuring */
```
Comment (remarcmij) [inline homework/index.js]: If I remove this there are still no ESLint errors/warnings.

File: homework/index.js (hunk 1 above)
Comment (gulyener) [reply homework/index.js]: You're right I must have missed this one. Thank you

File: homework/index.js (hunk 2 above)
Comment (gulyener) [reply homework/index.js]: Oh I didn't see your last reply on homework week 1. Now I understand thank you

**Pull request closed without merging**